.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
why we need this setting.json?

Reply:
Yes you can directly remove thi setting.json

Reply:
ok